run.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+if [ "${POSTGRES_ENV_POSTGRES_PASSWORD}" == "**Random**" ]; then
+        unset POSTGRES_ENV_POSTGRES_PASSWORD
+fi
+
+POSTGRES_HOST=${POSTGRES_PORT_5432_TCP_ADDR:-${POSTGRES_HOST}}
+POSTGRES_HOST=${POSTGRES_PORT_1_5432_TCP_ADDR:-${POSTGRES_HOST}}
+POSTGRES_PORT=${POSTGRES_PORT_5432_TCP_PORT:-${POSTGRES_PORT}}
+POSTGRES_PORT=${POSTGRES_PORT_1_3306_TCP_PORT:-${POSTGRES_PORT}}
+POSTGRES_USER=${POSTGRES_USER:-${POSTGRES_ENV_POSTGRES_USER}}
+POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-${POSTGRES_ENV_POSTGRES_PASSWORD}}
+
+[ -z "${POSTGRES_HOST}" ] && { echo "=> POSTGRES_HOST cannot be empty" && exit 1; }
+[ -z "${POSTGRES_PORT}" ] && { echo "=> POSTGRES_PORT cannot be empty" && exit 1; }
+[ -z "${POSTGRES_USER}" ] && { echo "=> POSTGRES_USER cannot be empty" && exit 1; }
+[ -z "${POSTGRES_PASSWORD}" ] && { echo "=> POSTGRES_PASSWORD cannot be empty" && exit 1; }
+[ -z "${POSTGRES_DB}" ] && { echo "=> POSTGRES_DB cannot be empty" && exit 1; }
+
+export PGPASSWORD="${POSTGRES_PASSWORD}"
+
+BACKUP_CMD="pg_dump -h ${POSTGRES_HOST} -p ${POSTGRES_PORT} -U ${POSTGRES_USER} -f /backup/\${BACKUP_NAME} ${POSTGRES_DB}"
+
+echo "=> Creating backup script"
+rm -f /backup.sh
+cat <<EOF >> /backup.sh
+#!/bin/bash
+MAX_BACKUPS=${MAX_BACKUPS}
+SLACK=${SLACK_WEBHOOK}
+BACKUP_NAME=dump_\$(date +%d-%m-%Y_%H_%M_%S).sql
+export PGPASSWORD="${POSTGRES_PASSWORD}"
+echo "=> Backup started: \${BACKUP_NAME}"
+if ${BACKUP_CMD} ;then
```

Comment:
did you test it? can we rely on the exit code?

Reply:
pg_dump return 1 if there is failure and 0 if succeeded. Tested